syslog-ng-ctl/syslog-ng-ctl.c

```diff
@@ -61,6 +61,34 @@ slng_reload(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
   return dispatch_command("RELOAD");
 }
 
+static gboolean config_options_start = FALSE;
+static gboolean config_options_stop = FALSE;
+
+static GOptionEntry internal_options[] =
+{
+  { "start", 's', 0, G_OPTION_ARG_NONE, &config_options_start, "start live collection of internal logs", NULL },
+  { "stop", 'x', 0, G_OPTION_ARG_NONE, &config_options_stop, "stop live collection of internal logs", NULL },
+  { NULL,           0,   0, G_OPTION_ARG_NONE, NULL,                         NULL,           NULL }
+};
+
+static gint
+slng_internal(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
+{
+  GString *cmd = g_string_new("");
+
+  if (config_options_start)
+    g_string_assign(cmd, "INTERLOGS START");
+  else if (config_options_stop)
+    g_string_assign(cmd, "INTERLOGS STOP");
+  else
+    return 1;
```

Comment:
We should not allow that start and stop are provided at the same time.

Reply:
@furiel : Are you talking about the command: `syslog-ng-ctl internal-logs --start --stop` ?
Can you suggest any solution for this?